specs/CustomDownload.md

```diff
@@ -0,0 +1,784 @@
+# Background
+<!-- TEMPLATE
+    Use this section to provide background context for the new API(s)
+    in this spec.
+
+    This section and the appendix are the only sections that likely
+    do not get copied into any official documentation, they're just an aid
+    to reading this spec.
+
+    If you're modifying an existing API, included a link here to the
+    existing page(s) or spec documentation.
+
+    For example, this section is a place to explain why you're adding this
+    API rather than modifying an existing API.
+
+    For example, this is a place to provide a brief explanation of some dependent
+    area, just explanation enough to understand this new API, rather than telling
+    the reader "go read 100 pages of background information posted at ...".
+-->
+We are redesigning the download experience by providing a DownloadStarting event, and exposing a Download object with all associated download metadata. With this you will be able to block downloads, save to a different path, and have access to the required metadata to build your own download UI.
+
+
+In this document we describe the updated API. We'd appreciate your feedback.
+
+# Description
+
+There are 3 parts to this API.
```

Comment:
Rather than duplicate the API listing up here, it might be better to summarize the 3 parts like the following and remove the 1,2,3 listing below that sort of duplicates info from the API Details section at the bottom.
```suggestion
There are 3 parts to this API: the CoreWebView2.DownloadStarting event that can be used to find out about or cancel new downloads, the Download class available from the DownloadStarting event that has up to date information on a download as it progresses, and the CoreWebView2Settings.DefaultDownloadDialog property that can be used to disable the default download UI.
```

Reply:
And also remove the 1,2,3 listing that is duplicating info from the API Details section at the bottom